### PR TITLE
added default score id for gnosis, changed implementation

### PR DESCRIPTION
**Pull request by milosstanisavljevic**

### CODE REVIEW

File: services/input-service.js [hunk 1]
```diff
@@ -150,11 +150,21 @@ class InputService {
...
-        return (
-            options.scoreFunctionId ??
-            this.config.scoreFunctionId ??
-            DEFAULT_PARAMETERS.SCORE_FUNCTION_ID
-        );
+        const blockchainName = this.getBlockchain(options).name.toLowerCase();
+
+        if (blockchainName.includes('hardhat1') || blockchainName.includes('otp')) {
+            return (
+                options.scoreFunctionId ??
```
[the remark this answers is not in the document]
Comment (milosstanisavljevic) [reply services/input-service.js:157]: there is a smarter way probably to read environment and blockchain name in input-service and based on that return value from constants , and to add this to constants.js :
```DEFAULT_PROXIMITY_SCORE_FUNCTIONS_PAIR_IDS = {
    "development": {"hardhat1:31337": 1, "hardhat2:31337": 2, "otp:2043": 1},
    "devnet": {
        "otp:2160": 1,
        "gnosis:10200": 2,
    },
    "testnet": {
        "otp:20430": 1,
        "gnosis:10200": 2,
    },
    "mainnet": {
        "otp:2043": 1,
        "gnosis:100": 2,
    },
}

File: services/input-service.js [hunk 2]
```diff
@@ -150,11 +154,12 @@ class InputService {
...
 
     getScoreFunctionId(options) {
-        return (
-            options.scoreFunctionId ??
-            this.config.scoreFunctionId ??
-            DEFAULT_PARAMETERS.SCORE_FUNCTION_ID
-        );
+        const environment =
+            options.environment ?? this.config.environment ?? DEFAULT_PARAMETERS.ENVIRONMENT;
+        console.log(environment);
```
Comment (Mihajlo-Pavlovic) [inline services/input-service.js:159]: Can you please remove console.log

Comment (milosstanisavljevic) [reply services/input-service.js:159]: good catch, sorry for that, will do right away

**Comment by NZT48** [comment]
For easier testing, all changes are merged to `feature/new-scoring-function` branch

**Pull request merged**